app/jobs/deposit_job.rb

```diff
@@ -74,17 +74,12 @@ def build_file_metadata(blobs_map)
       SdrClient::Deposit::Files::DirectUploadRequest.new(
         checksum: blob.checksum,
         byte_size: blob.byte_size,
-        content_type: clean_content_type(blob.content_type),
+        content_type: blob.content_type,
         filename: blob.filename.to_s
       )
     end
   end
 
-  def clean_content_type(content_type)
-    # ActiveStorage is expecting "application/x-stata-dta" not "application/x-stata-dta;version=14"
-    content_type&.split(';')&.first
-  end
-
```

Comment:
this logic is now handled in the new version of sdr-client, so we don't need it here anymore

Reply:
nice!  ok, yeah, i like moving this into sdr-client, despite my wishy-washy feedback on the sdr-client PR.  hadn't looked at this yet, so wasn't sure whether there'd be redundant processing (which i was more worried about for intelligibility than for performance reasons.